databricks/koalas/indexes/base.py

```diff
@@ -507,6 +507,29 @@ def to_numpy(self, dtype=None, copy=False) -> np.ndarray:
             result = result.copy()
         return result
 
+    def map(
+        self,
+        mapper: Union[dict, Callable[[Any], Any], pd.Series],
+        return_type: ks.typedef.Dtype = str,
+        na_action: Any = None,
+    ):
+        """
+        Use to change Index values
```

Comment:
How about leveraging [pandas docstring](https://github.com/pandas-dev/pandas/blob/v1.2.3/pandas/core/indexes/base.py#L5077-L5117)? It would be great to maintain the compatibility between Koalas APIs and pandas APIs.